jdl/parsing/jdl-parser.ts

```diff
@@ -352,7 +351,7 @@ module.exports = class JDLParser extends CstParser {
     });
   }
 
-  enumProp() {
+  enumProp(): any {
```

Comment:
I'm putting this comment here, but we can replace the any here by void

Reply:
and in this file

Reply:
Unfortunately void does not work.